dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynonfinancials/EuTaxonomyNonFinancialsFramework.kt

```diff
@@ -23,7 +23,7 @@ class EuTaxonomyNonFinancialsFramework : PavedRoadFramework(
     ),
     order = 2,
     enabledFeatures =
-    FrameworkGenerationFeatures.allExcept(FrameworkGenerationFeatures.UploadPage, FrameworkGenerationFeatures.QaModel),
+    FrameworkGenerationFeatures.allExcept(FrameworkGenerationFeatures.UploadPage),
```

Comment:
why was this deleted?

Reply:
Otherwise, the toolbox does not create the qa model